src/aiortc/rtcrtpreceiver.py

```diff
@@ -398,6 +399,14 @@ async def receive(self, parameters: RTCRtpReceiveParameters) -> None:
     def setTransport(self, transport: RTCDtlsTransport) -> None:
         self.__transport = transport
 
+    @property
+    def enabled(self) -> bool:
+        return self.__enabled
+
+    @enabled.setter
+    def enabled(self, enabled: bool) -> None:
+        self.__enabled = enabled
+
```

Comment:
I don't see any such property in the corresponding JS interface, so this seems odd?

Reply:
True, this could be `_enabled()` to indicate it's private.